internal/protocol/params.go

```diff
@@ -5,6 +5,9 @@ import "time"
 // DesiredReceiveBufferSize is the kernel UDP receive buffer size that we'd like to use.
 const DesiredReceiveBufferSize = (1 << 20) * 2 // 2 MB
 
+// DesiredSendBufferSize is the kernel UDP send buffer size that we'd like to use.
+const DesiredSendBufferSize = (1 << 20) * 2 // 2 MB
```

Comment:
Maybe a note why 1 << 20 (1 MiB) is actually 2MB here? (the linux kernel doubles this for bookkeeping I believe)

Reply:
There's a *2. It might be worth incorporating the doubling into the check in `setReceiveBuffer` though.

Reply:
oh, I missed the `*2` not sure how. Nvm.